appcenter-data/Data.js

```diff
@@ -69,6 +69,9 @@ Data.WriteOptions = class {
 };
 
 function convertTimestampToDate(result) {
+    if (!result === undefined || !result.lastUpdatedDate === undefined) {
```

Comment:
If `result` is either `null` or `undefined`, `!result` will return `true`, because `null` and `undefined` are falsey values.
`true` will never be `===` to `undefined`. same issue with `lastUpdateDate`

Reply:
Oops, fixed